libcaf_core/caf/event_based_mail.test.cpp

```diff
@@ -243,6 +244,83 @@ TEST("send request message") {
       expect<exit_msg>().to(dummy);
     }
   }
+  SECTION("using .to_observable for the response") {
+    SECTION("valid response") {
+      auto dummy = sys.spawn([]() -> behavior {
+        return {
+          [](int value) { return value * value; },
+        };
+      });
+      auto err = std::make_shared<error>();
+      auto result = std::make_shared<int>(0);
+      SECTION("valid response") {
+        self->mail(3)
+          .request(dummy, infinite)
+          .as_observable<int>()
```

Comment:
Can this infer the type for mail sent to a typed actor?

Reply:
Sure, the test for that is below (line 306).

Reply:
I missed that, thanks! 🙈